core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1310,6 +1318,7 @@ public String toString() {
               + "losing state in case of power loss or host failure. Use MEMORY for "
               + "optimal performance, but no state persistence across cluster restarts.")
           .setDefaultValue("DISK")
+          .setScope(Scope.MASTER)
```

Comment:
Add consistency check level also be warn?

Reply:
I think we might want to support different storage tiers across cluster.

Reply:
Oh, i saw it as the scope of `MASTER_EMBEDDED_JOURNAL_PORT`, never mind,